### ISSUE TITLE
Is publish method actually using HTTP or is it using HTTPS?

### ISSUE THREAD

**Issue by shua123** [socrata/datasync#104]
Is publish method actually using HTTP or is it using HTTPS?

**Comment by levyj** [comment]
If you change anything, please retain backwards compatibility to any flags that refer to HTTP.  Otherwise, it creates a barrier to upgrading DataSync.  Thanks.

**Comment by shua123** [comment]
In 1.6 the GUI was changed to no longer mention HTTP in the update method selection so I changed the title of this issue as the original title is no longer reflective of the question.

The question still remains though on what exactly DataSync is using. Is it HTTP or HTTPS or both? WIth [increasing emphasis](https://18f.gsa.gov/2015/06/08/the-us-government-is-moving-to-https-everywhere/) on the use of HTTPS use in government, DataSync documentation should clearly explain what protocols, processes, & methods are in use.
